charts/ups-broker/templates/broker-deployment.yaml

```diff
@@ -27,6 +27,14 @@ spec:
         args:
         - --port
         - "8080"
+        {{- if .Values.tls.cert}}
+        - --tlsCert
+        - "{{ .Values.tls.cert }}"
+        {{- end}}
+        {{- if .Values.tls.key}}
```

Comment:
not 100% sure on this, but iirc this will collapse `--tlsKey` onto the same line as `--tlsCert`. The solution would be to rewrite this as `{{ if .Values.tls.key -}}`. Same with the above `if` as well.